src/the-bureau/about-director/index.html

```diff
@@ -116,8 +116,9 @@ <h4 class="u-mb5">
                     </div>
                 </div>
             </div>
-            <div class="content-l_col
-                        content-l_col-1-2">
+            <div data-qa-hook="director-speaking-info"
+                class="content-l_col
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/8555b964c486de0b03084f0e58e3f97f44f7727a